src/graphorm/index.ts

```diff
@@ -187,6 +187,28 @@ const obj = new GraphORM({
               .andWhere(`${childAlias}."sourceId" = "${parentAlias}".id`),
         },
       },
+      owners: {
+        select: (ctx, alias, qb) => {
```

Comment:
We can actually combine these 2 in. a function like we do for `existsByUserAndPost` so it's easier to manipulate.

Suggestion: getMembersByRole? so we can pass a role.